### PR TITLE
Added flesh heap for int and agi

### PR DESCRIPTION
**Pull request by Wouterz90**
As requested in this [thread](https://github.com/LegendsOfDota/LegendsOfDota/issues/334) I have added the flesh heap versions for int and agi. It is basically a copy from the one in the spelllibrary, but with different stats. I have added a check to see if the target is not an illusion.

image

### CODE REVIEW

**Comment by Jexah** [comment]
Should these skills also have magic resist? Perhaps a different bonus instead? Just a thought.

**Comment by darklordabc** [comment]
My thoughts on the missing magic resist was that its okay as is. I tend to think that custom abilities should be underpowered when they are first released so they dont upset the meta too much, and then maybe gradually increased after a period of time.

**Comment by Jexah** [comment]
@Wouterz90 Also the reference to Global Break in `abilities.kv`, thanks.

**Comment by Wouterz90** [comment]
Did that and also changed the spell names to your standards as I should have done in the first place.

File: src/scripts/vscripts/lua_spells/flesh_heap_agi.lua [hunk 1]
```diff
@@ -0,0 +1,53 @@
+flesh_heap_agi = class({})
+
+LinkLuaModifier( "modifier_flesh_heap_agi", "lua_spells/modifiers/modifier_flesh_heap_agi.lua" ,LUA_MODIFIER_MOTION_NONE )
```
Comment (ash47) [inline src/scripts/vscripts/lua_spells/flesh_heap_agi.lua]: This file location makes NO SENSE. Please change it to be inline with the other spells, we aren't just going to create a random directory for every spell at this stage:

LegendsOfDota\src\scripts\abilities\

File: src/scripts/vscripts/lua_spells/modifiers/modifier_flesh_heap_agi.lua [hunk 2]
```diff
@@ -0,0 +1,41 @@
+--Taken from the spelllibrary, credits go to valve
+
+modifier_flesh_heap_agi = class({})
```
Comment (ash47) [inline src/scripts/vscripts/lua_spells/modifiers/modifier_flesh_heap_agi.lua]: Again, I don't like this "lua_spells" business, If you are going to have a lua based modifier, make a new folder "modifier" in the existing "abilities" folder

**Comment by ash47** [comment]
Review the comments I've made on the code, make the changes, etc.

**Comment by Wouterz90** [comment]
Thanks for checking this, I have never seen lua scripts outside the vscripts folder before, so I didn't see this coming.

**Comment by darklordabc** [comment]
This doesn't seem to be working right now. 
I tested it and the abilities does not have any tooltips, and when you try to start game with the abilities this happens.
image

**Comment by Wouterz90** [comment]
I am an idiot, by changing flesh_heap into pudge_flesh_heap I didn't change the classes, should sync the map instead of manually editing both

**Comment by darklordabc** [comment]
I downloaded the "master" branch on your repository and it doesn't seem to work. I can see the icons and choose them as skills, but tool-tips are missing and an error spits out at start of game. Perhaps I downloaded the wrong branch? 
http://hastebin.com/kovinuxuqo.tex

**Comment by ash47** [comment]
I need to test this in game before I can merge, since it's not touching my code, the syntax in the ability files doesn't matter too much, so, I will allow it for now.

**Comment by ash47** [comment]
Your effort going to waste, doesn't bother be in the slightest.

**Comment by Wouterz90** [comment]
I wouldn't expect anything else but this decision wasn't mine. I'm sorry to have wasted your time.

**Comment by Jexah** [comment]
If you want to keep contributing, there is nothing stopping you, just wait until the PayPal charge-back timeout has finished.

**Comment by ash47** [comment]
So, it got to the stage where I was actually getting ready to merge this
stuff, so, pretty funny, after 40 comments asking for this, you finally get
your way and now "too late"
On 25 Jun 2016 6:38 PM, "Jeremy Short" <notifications@github.com> wrote:

If you want to keep contributing, there is nothing stopping you, just wait
until the PayPal charge-back timeout has finished.

—
You are receiving this because you commented.

Reply to this email directly, view it on GitHub
<https://github.com/LegendsOfDota/LegendsOfDota/pull/345#issuecomment-228523985>,
or mute the thread
<https://github.com/notifications/unsubscribe/ADlKDoycoFxTip2M4rW3nur_u9Hzg6fBks5qPOkAgaJpZM4Izmto>
.

**Comment by ash47** [comment]
My own personal backup of this: https://github.com/LegendsOfDota/LegendsOfDota/tree/fleshheap_int

**Pull request closed without merging**